src/random.cpp

```diff
@@ -9,8 +9,16 @@
 #include <support/cleanse.h>
 #ifdef WIN32
 #include <compat.h> // for Windows API
-#include <wincrypt.h>
+#if defined(_MSC_VER) && defined(_WIN32_WINNT) && _WIN32_WINNT>=0x0601
+#include <bcrypt.h>
```

Comment:
so with this, it will still use the deprecated function with mingw?
(which is used for the distributed binaries)